src/core/Table/TableCell.tsx

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Cell, CellProps, TableInstance } from 'react-table';
+import cx from 'classnames';
+import { getCellStyle } from './utils';
+import { SubRowExpander } from './SubRowExpander';
+import { SELECTION_CELL_ID } from './hooks';
+import { DefaultCell } from './cells';
+
+export type TableCellProps<T extends Record<string, unknown>> = {
+  cell: Cell<T>;
+  cellIndex: number;
+  isDisabled: boolean;
+  tableHasSubRows: boolean;
+  tableInstance: TableInstance<T>;
+  expanderCell?: (cellProps: CellProps<T>) => React.ReactNode;
+};
+
+export const TableCell = <T extends Record<string, unknown>>(
```

Comment:
Why this is not in the cells folder?

Reply:
I wanted to avoid confusion between this component and `DefaultCell`.